DHI inline patch (druid 37.0): bump vertx-core for CVE-2026-6860

Advisory requires vertx-core 4.5.27+:
https://scout.docker.com/v/CVE-2026-6860

diff --git a/extensions-core/kubernetes-overlord-extensions/pom.xml b/extensions-core/kubernetes-overlord-extensions/pom.xml
index 1111111..2222222 100644
--- a/extensions-core/kubernetes-overlord-extensions/pom.xml
+++ b/extensions-core/kubernetes-overlord-extensions/pom.xml
@@ -30,7 +30,7 @@
   </parent>
 
   <properties>
-    <vertx.version>4.5.24</vertx.version>
+    <vertx.version>4.5.28</vertx.version>
   </properties>
 
   <dependencies>
